features/messages.feature

```diff
@@ -0,0 +1,19 @@
+@api @d8 @arb
```

Comment:
Lol, I was using that for local testing as the behat command seems broken when trying to run line numbers of features combined with `--profile` :)